staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -839,7 +857,7 @@ func (qs *queueSet) finishRequestLocked(r *request) {
 
 		// When a request finishes being served, and the actual service time was S,
 		// the queue’s start R is decremented by (G - S)*width.
-		r.queue.virtualStart -= (qs.estimatedServiceSeconds - S) * float64(r.Seats())
+		r.queue.virtualStart -= (qs.estimatedServiceSeconds - S) * float64(r.workEstimate.InitialSeats)
```

Comment:
This PR changes _some_ of the updates to `queue::virtualStart` to use units of seat-microseconds, leaving others (like this one) in units of seat-seconds.  I see no need for this PR to introduce any switch to microseconds, since we are already contemplating an additional PR to change how work is represented.

Reply:
yes, in anticipation of https://github.com/kubernetes/kubernetes/pull/105322, once #105322 gets merged I will rebase.